editor/js/Loader.js

```diff
@@ -572,4 +582,45 @@ var Loader = function ( editor ) {
 
 	}
 
+	function isGltf1( contents ) {
+
+		var resultContent;
+
+		if ( typeof contents === 'string' ) {
+
+			resultContent = contents;
+
+		} else {
+
+			var magic = THREE.LoaderUtils.decodeText( new Uint8Array( contents, 0, 4 ) );
+
+			if ( magic === 'glTF' ) {
+
+				try {
+
+					extensions[ EXTENSIONS.KHR_BINARY_GLTF ] = new GLTFBinaryExtension( contents );
```

Comment:
`extensions` is a copy/paste artifact, I imagine? In fact this line will [throw a specific error if the asset has a version `<=2`](https://github.com/mrdoob/three.js/blob/dev/examples/js/loaders/GLTFLoader.js#L387-L391). If it parses successfully it is glTF 2.0.

Reply:
Oops, I see now GLTFBinaryExtension is defined in both LegacyGLTFLoader and GLTFLoader, within closures... that seems like an issue, will this code run correctly?

Reply:
Okay, will take that section out entirely.